python/ray/dashboard/client/src/components/ActorTable.component.test.tsx

```diff
@@ -232,7 +232,7 @@ describe("ActorTable", () => {
       - Used memory: Actor 1 memory > Actor 2 memory --> Actor 1 row before Actor 2 row
       - Uptime: Actor 2 uptime < Actor 1 uptime --> Actor 2 row before Actor 1 row
       - GPU Utilization: Actor 1 GPU > Actor 2 GPU --> Actor 1 row before Actor 2 row
-      - GRAM Utilization: Actor 2 GRAM > Actor 1 GRAM --> Actor 2 row before Actor 1 row
```

Comment:
I never questioned the term `GRAM`, it was there since the beginning of the dashboard, but I guess `VRAM` is definitely the correct and standard term.

However, can we avoid making this change for now? I want to sync with the broader team / maintainers to see if we want to apply this new terminology across the board (including in metrics, APIs, etc).